coral-spark/src/test/java/com/linkedin/coral/spark/CoralSparkTest.java

```diff
@@ -191,8 +191,8 @@ public void testNoUdf() {
   public void testLateralView() {
     RelNode relNode = TestUtils.toRelNode(
         String.join("\n", "", "SELECT a, t.ccol", "FROM complex", "LATERAL VIEW explode(complex.c) t as ccol"));
-    String targetSql = String.join("\n", "SELECT complex.a, t1.ccol",
-        "FROM default.complex LATERAL VIEW EXPLODE(complex.c) t1 AS ccol");
+    String targetSql =
+        "SELECT complex.a, t00.ccol\n" + "FROM default.complex LATERAL VIEW EXPLODE(complex.c) t00 AS ccol";
```

Comment:
Just for my understanding, could you explain why `t1` will become `t00`?

Reply:
It's related to the table alias generation code.  In the earlier code, the aliases that got generated were `t0` and `t1` since they are both based on the basename `t`.  In the new code, the first table alias generated `t0` is used to generate the second table alias so an extra `0` is added to the name of the table.

The main reason for this change is the simplification of the use of `SELECT` and `AS` nodes in the handling of Explode function.  The old code was doing: `AS`, `SELECT`, `AS`, while the new code use a single `AS`.